src/client/src/views/Analytics/AffectedPopulationProvince.vue

```diff
@@ -1,11 +1,99 @@
 <template>
-  <h1>Hello from Affect Population - Province component</h1>
+  	<div class="mt-5">
+		<b-form inline @submit="FormSubmission">
+			<b-input v-model="Country" required placeholder="Country"></b-input>
+			<b-input
+				class="ml-3"
+				v-model="Province"
+				required
+				placeholder="State/Provinces"
+			></b-input>
+			<b-button type="submit" variant="primary" class="ml-5">Search</b-button>
+		</b-form>
+
+    <div v-if="showChart" class="mt-5">
+			<!-- this will only render if showChart is a truthy value -->
+			<b-row>
+				<!-- row contains donut chart & selection -->
+				<b-col>
+					<DonutChart v-bind:data="chartData" />
+					<!-- will display whatever bind to it -->
+				</b-col>
+			</b-row>
+		</div>
+  	</div>
+    
 </template>
 
 <script>
+import Services from '../../Services/Services';
+import DonutChart from '../../components/DonutChart';
+
+
 export default {
+  name: 'AffectedPopulationProvince',
+	data() {
+		return {
+			//Changes according to the input field (user inputs)
+			Country: null,
+			Province: null,
+			chartData: null,
+			showChart: false,
+		};
+	},
+	methods:{
+    async FormSubmission(e) {
+			e.preventDefault();
+			await this.GetData('1'); //first submit gets confirmed as default
```

Comment:
I think you can put all of your logic to make the request to the backend in one function, here in "FormSubmission." so you can pretty much move the logic of your "GetData" function to this "FormSubmission" function. Remove line 47 because we don't need the "GetData" function anymore. Don't remove line 46 because that's what prevents the page to reload once the user submits the form.